kubernetes/loculus/values_preview_server.yaml

```diff
@@ -42,6 +42,7 @@ sequenceFlagging:
     repository: curation_reports
     issueTemplate: sequence-metadata-issue.md
 auth:
+  verifyEmail: false
```

Comment:
hmm this is not working in the preview

Reply:
<img width="531" alt="image" src="https://github.com/user-attachments/assets/5b81d741-3603-40ef-b56e-de11c84910b5" />


Do you also get this when trying to register?

Reply:
Right sorry - so when I posted this there were two conflicting things specified: https://github.com/loculus-project/loculus/pull/3804/commits/11bf4405cb3179bdce99f1b9ecd63ffc9adbb41b and I was just getting a normal verification email. Then after I fixed that you got that error and I got the same when I tried - I think that was because we ran out of resources and the keycloak DB couldn't run. Everything is fine now.